mobile/lib/presentation/widgets/people/person_edit_name_modal.widget.dart

```diff
@@ -50,16 +73,106 @@ class _DriftPersonNameEditFormState extends ConsumerState<DriftPersonNameEditFor
+          children: [
+            TextFormField(
+              autofocus: true,
+              controller: _formController,
+              decoration: InputDecoration(
+                hintText: 'add_a_name'.tr(),
+                border: const OutlineInputBorder(borderRadius: BorderRadius.all(Radius.circular(8))),
+              ),
+              onChanged: (value) => _filterPeople(people, value),
+              onTapOutside: (event) => FocusScope.of(context).unfocus(),
+            ),
+            curatedPeople.when(
+              data: (p) {
+                people = p;
+                return AnimatedSize(
+                  duration: const Duration(milliseconds: 200),
+                  child: SizedBox(
+                    width: double.infinity,
+                    child: _filteredPeople.isEmpty
+                        // Tile instead of a blank space to avoid horizontal layout shift
+                        ? LargeLeadingTile(
+                            leading: const SizedBox.shrink(),
+                            onTap: () {},
+                            title: const SizedBox.shrink(),
+                            disabled: true,
+                          )
+                        : Container(
+                            margin: const EdgeInsets.only(top: 8),
+                            decoration: const BoxDecoration(borderRadius: BorderRadius.all(Radius.circular(8))),
+                            child: Column(
+                              mainAxisSize: MainAxisSize.min,
+                              children: _filteredPeople.map((person) {
+                                return PersonTile(
+                                  isSelected: false,
+                                  onTap: () {
+                                    if (!mounted) return;
+                                    setState(() {
+                                      _formController.text = person.name;
+                                    });
+                                    _formController.selection = TextSelection.fromPosition(
+                                      TextPosition(offset: _formController.text.length),
+                                    );
+                                    onMerge(context: context, person: widget.person, mergeTarget: person);
+                                  },
+                                  personName: person.name,
+                                  personId: person.id,
+                                );
+                              }).toList(),
+                            ),
+                          ),
+                  ),
+                );
+              },
+              loading: () => const Center(child: CircularProgressIndicator()),
+              error: (err, stack) {
+                Logger('PersonEditNameModal').warning('Error loading people for name edit modal', err, stack);
+                return Center(child: Text('Error loading people for name edit modal: $err'));
```

Comment:
Open question for the more involved, whats the philosophy in immich: Should I display the error to the user (thus translate) or "hide" it in the logs?